src/index.css

```diff
@@ -44,3 +44,21 @@ body {
 .operand {
   background: rgb(250, 120, 34);
 }
+
+.wrapper {
+  display: flex;
+  justify-content: flex-start;
+  margin-top: 4rem;
+  gap: 10rem;
+}
```

Comment:
[ Suggested Changes ]

- [ ] It would be nice if you could adjust things to be in the center of the page instead of being on the left. You could achieve this in many other ways but simply, you could change the flex-start to `justify-content: space-around;`. flex-start makes things be at the left but, with the space-around style, things will have equal spacing around elements both left and right and which in turn gets things in the center. 👍🏽 

NOTE:
This is not a major change but I highly recommend you change the flex-start to `justify-content: space-around;` so your app looks nice. 👍🏽